dts/bindings/led_strip/ws2812-gpio.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024, TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+include: ws2812.yaml
+
+description: |
+  Common definition GPIO based WS2812 node
+
+properties:
+  gpios:
+    type: phandle-array
```

Comment:
The `gpios` name may be treated specially during devicetree processing.
In particular, since it includes processing to link with `*-names`, even when specifying a single GPIO pin, there are many places where the name is `*gpios` in the phandle-array type. It also follows this convention.

https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/dts/python-devicetree/src/devicetree/edtlib.py#L1757C17-L1757C33

Reply:
Thats good enough for me.

Reply:
Just to clarify: `phandle-array` is not for storing an array of phandles (that would be `type: phandles`), it is for storing a list of phandles and specifiers (e.g. GPIO port, pin, and flags).

Using a `phandle-array` type here seems correct to me.

See https://docs.zephyrproject.org/latest/build/dts/bindings-syntax.html#type for further information of the various devicetree types.